src/components/mixins/player.js

```diff
@@ -628,6 +628,48 @@ export const playerMixin = {
       }
     },
 
+    goPreviousDrawing() {
+      try {
+        this.clearCanvas()
+        const annotation_time =
+          Number(this.getPreviousAnnotationTime(this.currentTimeRaw).frame) - 1
+        if (this.isFullMode) {
+          const nextDrawingTime = annotation_time / this.fps
+          this.setFullPlayerTime(nextDrawingTime)
+        } else {
+          this.rawPlayer.setCurrentTimeRaw(annotation_time / this.fps)
+          this.onProgressChanged(annotation_time, true)
+        }
+        if (this.isComparing) {
+          this.syncComparisonViewer()
+        }
+      } catch {
+        console.log('should never happen')
```

Comment:
Improve error log to aid debugging:
```suggestion
      } catch (err) {
        console.error('should never happen', err)
```